lib/resolve.js

```diff
@@ -1,3 +1,5 @@
+const minimist = require('minimist');
```

Comment:
Looks good, i think better use `yargs` instead `minimist`, he is deprecated and abandoned (it was said contributor of `minimist`)

Reply:
minimist is WAY smaller than yargs and it's very stable for this purpose in its last release (1.2.0). keeping this is OK